Disable JIT for non-Intel architectures since it only works on Intel.
--- libzpaq.cpp.orig	2016-08-17 16:49:18.000000000 -0500
+++ libzpaq.cpp	2023-07-07 02:46:59.000000000 -0500
@@ -28,6 +28,10 @@
 #include <vector>
 #include <stdio.h>
 
+#if !defined(__x86_64__) && !defined(__i386__)
+#define NOJIT
+#endif
+
 #ifdef unix
 #ifndef NOJIT
 #include <sys/mman.h>
